coordinator/src/main/java/org/apache/uniffle/coordinator/SimpleClusterManager.java

```diff
@@ -285,6 +285,29 @@ public List<ServerNode> getUnhealthyServerList() {
     return Lists.newArrayList(unhealthyNodes);
   }
 
+  @Override
+  public List<ServerNode> getDecommissioningServerList() {
+    List<ServerNode> decommissioningNodes = Lists.newArrayList();
```

Comment:
Can we abstract a method here, because it is very similar to getDecommissionedServerList? 🤔️